pawnShop/src/main/java/com/example/pawnShop/Controller/ProductTypeController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.pawnShop.Controller;
+
+import com.example.pawnShop.Dto.Product.ProductTypeDto;
+import com.example.pawnShop.Service.Contract.ProductTypeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/product_type")
+public class ProductTypeController {
+    private final ProductTypeService productTypeService;
+
+    @GetMapping("/get_all")
+    public ResponseEntity<List<ProductTypeDto>> getAllProductsTypes(){
+        var productsTypes = productTypeService.getAllProductsTypes();
+
+        return ResponseEntity.ok(productsTypes);
+    }
+    @GetMapping("/get_by_id")
+    public ResponseEntity<ProductTypeDto> getProductTypeById(@RequestParam UUID id){
+        var productType = productTypeService.getProductTypeById(id);
+
+        return ResponseEntity.ok(productType);
+    }
+    @PostMapping("/add")
```

Comment:
Hi again, Toshe @Todorov135 
it's a good convention to use only one endpoint and change only the specific request options, example:
requestById -> "/products/{uuid}"
save -> "products" (changed to POST)
...

Reply:
HINT:
This endpoint should be changed to @PostMapping("/products")